packages/ubuntu_provision/test/test_utils.dart

```diff
@@ -84,3 +86,31 @@ class _Dummy {} // ignore: unused_element
   MockSpec<SvgPicture>(),
 ])
 class _Dummy2 {} // ignore: unused_element
+
+class FakeAssetBundle extends CachingAssetBundle {
```

Comment:
This is the same as in the xdg keyboard service test. I didn't move it into a shared package like ubuntu_utils and just copied it instead, since we'll hopefully get rid of the xdg keyboard service soon.